lib/bronto/contact.rb

```diff
@@ -1,17 +1,20 @@
 module Bronto
   class Contact < Base
-    attr_accessor :email, :fields, :list_ids, :created, :modified, :status, :num_opens, :num_clicks, :num_sends
+    attr_accessor :email, :mobile_number, :sms_keyword_ids, :fields, :list_ids, :created, :modified, :status, :num_opens, :num_clicks, :num_sends
+
     # Finds contacts based on the `filter` (Bronto::Filter object).
     # * `page_number` is the page of contacts to request. Bronto doesn't specify how many contacts are returned per page,
     #    only that you should keep increasing the number until no more contacts are returned.
     # * `fields` can be an array of field IDs or an array of Field objects.
     # * `include_lists` determines whether to include the list IDs each contact belongs to.
-    def self.find(filter = Bronto::Filter.new, page_number = 1, fields = nil, include_lists = false, api_key = nil)
+    # * `include_keywords` determines whether to include the SMS Keyword IDs each contact belongs to.
+    def self.find(filter: Bronto::Filter.new, page_number: 1, fields: nil, include_lists: false, include_keywords: false, api_key: nil)
```

Comment:
This introduces a breaking change I don't feel comfortable including now. Can you revert back to positional arguments?